### PR TITLE
Add validates_presence_of_liquid_tag implementation 

### PR DESCRIPTION
**Pull request by melashu**
# In this milestone I added the following thing

- Add validates_presence_of_liquid_tag implementation 
- Add test for validates_presence_of_liquid_tag

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,8 +21,17 @@ In it's simplest form:
 class Article < ActiveRecord::Base
   validates_liquid_of :content
   validates_presence_of_liquid_variable :content, variable: 'very_important_variable'
+  validates_presence_of_liquid_filter :content, filter: 'your_filter', max: "1"
```
Comment (espen) [inline README.md]: max should be integer. here and also other places.

File: spec/liquid_validations_spec.rb
Comment (espen) [inline spec/liquid_validations_spec.rb:1]: please add tests for:
* valid test when including filter
* valid test when using more complex filter: {% filtername 'foobar', 'data-required="true"' %}
* invalid test when filter is used but used less than max count.
* invalid test when using multiple other filters but not the one that is used in the validation.

Also try to include a bit more text in the content for better test coverage.

**Review by espen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (melashu) [reply README.md]: Thanks

**Comment by espen** [comment]
Looks there is something wrong with these commits. There is a lot of changes that is not really changes. Also there are changes to many files (license, travis etc) that are not edited. Maybe some whitespace issue?

Also, my mistake but I called this filters. The correct terminology is tag. It is a Liquid Tag. So you can rename all references for filter to tag. Like validates_presence_of_liquid_tag

File: README.md [hunk 2]
```diff
@@ -21,15 +21,15 @@ In it's simplest form:
...
-`validates_presence_of_liquid_filter` takes required `filter` and `max` property
+`validates_presence_of_liquid_tag` takes required `filter` and `max` property
 
 ``` ruby
-validates_presence_of_liquid_filter :content, 
-                                    filter: 'head_content', 
-                                    max: '1'
+validates_presence_of_liquid_tag :content, 
+                                  filter: 'head_content', 
+                                  max: '1'
```
Comment (espen) [inline README.md]: should be integer

**Pull request closed without merging**